projects/addon-editor/components/toolbar/toolbar.template.html

```diff
@@ -232,19 +232,33 @@
         *ngIf="enabled(TuiEditorTool.Color) || enabled(TuiEditorTool.Hilite)"
         class="t-block"
     >
-        <tui-text-color
+        <div
             *ngIf="enabled(TuiEditorTool.Color)"
             tuiItem
             class="t-tool t-wrapper"
-            [colors]="colors"
-        ></tui-text-color>
+        >
+            <div #overrideTool>
+                <ng-content select="color"></ng-content>
+            </div>
+            <tui-text-color
+                *ngIf="!overrideTool.children.length"
```

Comment:
What is the benefit of this, instead of disabling built-in color tools and adding your own as custom?

Reply:
fixed